stockflux-securities-master/src/components/SecuritiesTable.js

```diff
@@ -4,12 +4,12 @@ import "./SecuritiesTable.css";
 import { Link } from "react-router-dom";
 import { getSecuritiesData } from "../services/SecuritiesService";
 import AddSecurityButton from "./AddSecurityButton";
-import ErrorMessage from "./ErrorMessage";
+import Alert from "./Alert";
 
 const SecuritiesTable = () => {
   const [securitiesData, setSecuritiesData] = useState([]);
   const [isLoading, setIsLoading] = useState(true);
-  const [errorMessage, setErrorMessage] = useState(null);
+  const [errorMessages, setErrorMessages] = useState([null]);
```

Comment:
Why is the default state an array of null? Shouldn't it just be an empty array?

Reply:
Okay done